tajo-core/src/main/java/org/apache/tajo/engine/function/datetime/AddDays.java

```diff
@@ -52,8 +56,16 @@ public AddDays() {
   }
 
   @Override
+  public void init(OverridableConf context, FunctionEval.ParamType[] types) {
+    if (!hasTimeZone()) {
+      setTimeZone(context.getConf().getSystemTimezone());
```

Comment:
```add_days()``` and ```add_months()``` seem to be separately implemented for each type of ```date``` and ```timestamp```. Here is a case which looks a bug. The results of ```add_days()``` of timestamp and date types are expected to be same, but is different.
```
default> \d test

table name: default.test
table uri: hdfs://localhost:7020/tajo/warehouse/default/test
store type: TEXT
number of rows: 1
volume: 40 B
Options:
	'timezone'='Asia/Seoul'
	'text.delimiter'='|'

schema: 
t	TIME
ts	TIMESTAMP
d	DATE

default> select * from test;
t,  ts,  d
-------------------------------
01:01:23,  2015-05-01 01:01:23,  2015-05-01
02:02:33,  2016-05-01 02:02:33,  2016-05-01
(2 rows, 0.019 sec, 0 B selected)
default> select add_days(ts, 5) from test;
?add_days
-------------------------------
2015-05-06 01:01:23
2016-05-06 02:02:33
(2 rows, 0.026 sec, 0 B selected)
default> select add_days(d, 5) from test;
?add_days
-------------------------------
2015-05-05 10:00:00
2016-05-05 10:00:00
(2 rows, 0.023 sec, 0 B selected)
```